nat-lab/tests/config.py

```diff
@@ -22,13 +22,6 @@ def get_root_path(path: str) -> str:
 DOCKER_NETWORK_MASK = "255.255.0.0"
 STUN_SERVER = "10.0.1.1"
 
-# Same as defined in `libtelio/nat-lab/provision_windows.ps1`
-stun.STUN_BINARY_PATH_WINDOWS = "C:/workspace/stunserver/release/stunclient.exe"
-stun.STUN_BINARY_PATH_MAC = "/Users/vagrant/stunserver/stunclient"
-
-IPERF_BINARY_MAC = "/Users/vagrant/iperf3/iperf3"
-IPERF_BINARY_WINDOWS = "C:\\workspace\\iperf3\\iperf3.exe"
```

Comment:
I think those should stay where they were. But I would change stun binary paths to constants in config.py and import config in stun.py. Like it was done for IPERF and is done for any other path from this file

Reply:
moved